launch-notes.md

```diff
@@ -0,0 +1,208 @@
+# Build
```

Comment:
Let's keep it as it is for now. Antoine will come up with a all-in script for the full network at some point

Reply:
I'm fine with wherever this ends up. I wanted to write it in a place we could all collaborate so we could learn together about the deployment. Parts of this will definitely need to be appropriated for docs for third-party full nodes and collators.